Library/Homebrew/tap.rb

```diff
@@ -567,3 +592,30 @@ def alias_file_to_name(file)
     file.basename.to_s
   end
 end
+
+class TapConfig
+  attr_reader :tap
+
+  def initialize(tap)
+    @tap = tap
+  end
+
+  def [](key)
+    return unless tap.git?
+    return unless Utils.git_available?
+
+    tap.path.cd do
+      Utils.popen_read("git", "config", "--local", "--get", "homebrew.#{key}").chuzzle
+    end
+  end
+
+  def []=(key, value)
+    return unless tap.git?
+    return unless Utils.git_available?
+
+    tap.path.cd do
+      safe_system "git", "config", "--local", "--add", "homebrew.#{key}", value.to_s
```

Comment:
i.e. `config["foo"]="a"; config["foo"]="b"` should overwrite value a to b instead of add value b.